src/m365/base/PowerAppsCommand.spec.ts

```diff
@@ -27,10 +29,17 @@ describe('PowerAppsCommand', () => {
 
   before(() => {
     sinon.stub(telemetry, 'trackEvent').returns();
+    sinon.stub(accessToken, 'isAppOnlyAccessToken').returns(false);
+    auth.connection.accessTokens[auth.defaultResource] = {
+      expiresOn: 'abc',
+      accessToken: 'abc'
+    };
+    auth.connection.active = true;
```

Comment:
If you add the `auth.connection.active` here, let's get rid of it in the existing tests then.